Exiled.API/Features/Toys/Light.cs

```diff
@@ -0,0 +1,120 @@
+// -----------------------------------------------------------------------
+// <copyright file="Light.cs" company="Exiled Team">
+// Copyright (c) Exiled Team. All rights reserved.
+// Licensed under the CC BY-SA 3.0 license.
+// </copyright>
+// -----------------------------------------------------------------------
+
+namespace Exiled.API.Features.Toys
+{
+    using AdminToys;
+    using Mirror;
+    using UnityEngine;
+
+    /// <summary>
+    /// A wrapper class for <see cref="AdminToys.LightSourceToy"/>.
+    /// </summary>
+    public class Light
+    {
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Light"/> class.
+        /// </summary>
+        public Light()
+        {
+            Base = Object.Instantiate(ToysHelper.LightBaseObject);
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="Light"/> class from a <see cref="AdminToys.LightSourceToy"/>.
+        /// </summary>
+        /// <param name="toy">The <see cref="AdminToys.LightSourceToy"/> to be wrapped.</param>
+        public Light(LightSourceToy toy)
+        {
+            Base = toy;
+        }
+
+        /// <summary>
+        /// Gets the base <see cref="AdminToys.LightSourceToy"/>.
+        /// </summary>
+        public LightSourceToy Base { get; }
+
+        /// <summary>
+        /// Gets or sets the position of the light.
+        /// </summary>
+        public Vector3 Position
+        {
+            get => Base.NetworkPosition;
```

Comment:
NetworkPosition will return the synced position of the object, not the real one always.